docker/Dockerfile

```diff
@@ -12,11 +13,17 @@ RUN apt-get update && apt-get install -y git ninja-build libglib2.0-0 libsm6 lib
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
 
-# Install mmsegmentation
 RUN conda clean --all
 
-RUN pip install mmcv-full==latest+torch1.6.0+cu101 -f https://download.openmmlab.com/mmcv/dist/index.html
+# Install MMCV
+ARG PYTORCH
+ARG CUDA
+ARG MMCV
+RUN pip install mmcv-full==${MMCV} -f https://download.openmmlab.com/mmcv/dist/cu${CUDA//./}/torch${PYTORCH}/index.html
```

Comment:
HI,when I build docker image i get a error

Error log:
Step 14/19 : RUN pip install mmcv-full==${MMCV} -f https://download.openmmlab.com/mmcv/dist/cu${CUDA//./}/torch${PYTORCH}/index.html
 ---> Running in fd7d2a76284c
/bin/sh: 1: Bad substitution
The command '/bin/sh -c pip install mmcv-full==${MMCV} -f https://download.openmmlab.com/mmcv/dist/cu${CUDA//./}/torch${PYTORCH}/index.html' returned a non-zero code: 2

it because ubuntu set dash shell as default bash shell,just like  /bin/sh link to /bin/dash but not /bin/bash.
So,I think you need try to avoid this different meanings,you can

RUN  /bin/bash -c 'pip install mmcv-full==${MMCV} -f https://download.openmmlab.com/mmcv/dist/cu${CUDA//./}/torch${PYTORCH}/index.htmli'

And i test its ok

Reply:
So , 
`RUN pip install mmcv-full==${MMCV} -f https://download.openmmlab.com/mmcv/dist/cu${CUDA//./}/torch${PYTORCH}/index.html`
should change to 
`RUN ["/bin/bash", "-c", "pip install mmcv-full==${MMCV} -f https://download.openmmlab.com/mmcv/dist/cu${CUDA//./}/torch${PYTORCH}/index.html"]`?

Reply:
Fixed, please check it again.

Reply:
OK,I build docker image success.And image content is right.